portal-test/src/com/liferay/portal/kernel/test/util/ServiceContextTestUtil.java

```diff
@@ -76,4 +76,18 @@ public static ServiceContext getServiceContext(
 		return serviceContext;
 	}
 
+	public static ServiceContext getServiceContext(
+			long userId, long groupId, long[] assetCategoryIds,
+			String[] assetTagNames)
+		throws PortalException {
+
+		ServiceContext serviceContext =
```

Comment:
Instead using ServiceContextTestUtil.getServiceContext(groupId, userId);, try that now, all the methods call ServiceContext getServiceContext(
			long userId, long groupId, long[] assetCategoryIds,
			String[] assetTagNames)

Reply:
true, thanks @cgoncas

Reply:
Adding a question here, when you said 
> one where everything is saved that have all the parameters:
> `ServiceContext getServiceContext(long userId, long groupId, long[] assetCategoryIds, String[] assetTagNames)`

you meant a method called by the rest, so it should include the `companyId` too, right? Please, see https://github.com/liferay/liferay-portal/blob/master/portal-test/src/com/liferay/portal/kernel/test/util/ServiceContextTestUtil.java#L65

Reply:
https://github.com/liferay-tango/liferay-portal/pull/889#issuecomment-780001167